### PR TITLE
Image Upload Dialog

### PR DESCRIPTION
**Pull request by pal03377**
This PR includes dialog components for uploading the user's profile picture, an institution profile picture and the institution background image.

It updates the already previously used package `material-file-dropzone` to the still somewhat maintained package `mui-file-dropzone`, which is also compatible with MUI 5.
In addition, it introduces a dependency on `react-image-crop` for image cropping.

The components can be found in `components/general/upload`. They behave like normal dialogs with an `open` and an `onClose` property. The `ProfileImageUploadDialog` is already integrated, so for example usage see `components/ProfileImage`.

This is how it looks like (colors are a bit broken by the screen recorder):
demo

### CODE REVIEW

**Comment by pal03377** [comment]
The upload itself does not currently work because https://github.com/jst-seminar-rostlab-tum/genomics.ai/pull/21 is not merged & not online, but it should in theory.

**Comment by pal03377** [comment]
Because I love demo videos, here's another one:

https://user-images.githubusercontent.com/9006596/165954176-cc09de27-3fb5-4aa4-9a91-a49ba6556c64.mp4

**Comment by pal03377** [comment]
Because I love demo videos, here's another one:

https://user-images.githubusercontent.com/9006596/165954176-cc09de27-3fb5-4aa4-9a91-a49ba6556c64.mp4

**Comment by pal03377** [comment]
> The upload itself does not currently work because #21 is not merged & not online, but it should in theory.

It's now actually battletested against a development server instance (which is also the new `BACKEND_ADDRESS` used in this PR). Thanks to @FabianNowak for providing that to me!

File: Frontend/src/components/ProfileImage/index.jsx [hunk 1]
```diff
@@ -1,18 +1,68 @@
...
-function ProfileImage({ sizePixels }) {
+import stringToColor from 'shared/utils/stringColor';
+import getProfile from 'shared/services/profile';
+
+function ProfileImage({ sizePixels, editable = false, overrideProfileImage = null }) {
+  const [user, setUser] = useState({});
+  async function loadUser() {
+    setUser(await getProfile());
+  }
+  useEffect(loadUser, [setUser]);
```
Comment (viganll) [inline Frontend/src/components/ProfileImage/index.jsx:15]: is this working? shouldn't it be like `useEffect(()=>{ loadUser()} ,[])`

Comment (pal03377) [reply Frontend/src/components/ProfileImage/index.jsx:15]: It is, I mean in a functional programming language like JS `()=>{ loadUser()}` and just `loadUser` are (almost) equivalent except for parameters, which there are none here, right? I think it is possible to leave out the `[setUser]` part, but it shouldn't hurt 🤔

**Pull request merged**